lib/cli.rb

```diff
@@ -215,6 +215,15 @@ def list_backups
     end
   end
 
+  desc 'show-backlog', 'Show backlog of board'
+  option 'board-id', desc: 'Id of Trello board', required: true
+  def show_backlog
+    puts 'Title'
+    TrelloWrapper.new(@@settings).board(options['board-id']).sprint_backlog_column.cards.each do |card|
```

Comment:
if I am not wrong, this should be `planning_backlog_column` and not `sprint_backlog_column`.

@cornelius wrote in https://github.com/openSUSE/trollolo/issues/45#issue-185365473:

> When doing sprint commitments we need a way to see the whole backlog in one view.

Normally when preparing the next Sprint in a team using Scrum, you have to decide how many task the team is able to do in the Sprint. That why I think this is in the planning board and not in the sprint one. @cornelius is that right?

**Note:** A sprint can be for example two week. The team decide at the start what they are going to work in this two week. @matheussbernardo if you think it would be useful to read more about how scrum works, please tell me and I will send you some material. 😉

Reply:
Also, what about writing this like:

```ruby
trello = TrelloWrapper.new(@@settings)
board = trello.board(board_id(options['board-id']))


board.planning_backlog_column.cards.each do |card|
```

More readable when splitting it 😉